cmd/tnf/fetch/fetch.go

```diff
@@ -156,7 +157,7 @@ func getOperatorCatalogSize() (size, pagesize uint) {
 	if err != nil {
 		log.Fatalf("Error in unmarshaling body: %v", err)
 	}
-	return aCatalog.Total, aCatalog.PageSize
+	return aCatalog.Total, operatorCatalogPageSize
 }
 
 func getOperatorCatalogPage(page, size uint) {
```

Comment:
You are not using the "size" value in the URL for the http GET, so you'll have only the default size (100) on each page, instead of the expected (500). Line 169 should be:
```
	url := fmt.Sprintf("%spage=%d&page_size=%d%s", operatorcatalogURL, page, size, filterCertifiedOperators)
```
Also, adding a log line would be nice, like:
```
	log.Infof("Getting operators page %d (size %d), url: %s", page, size, url)
```